crux-metrics/src/crux/metrics/dropwizard/cloudwatch.clj

```diff
@@ -0,0 +1,36 @@
+(ns crux.metrics.dropwizard.cloudwatch
+  (:require crux.metrics.dropwizard)
+  (:import [io.github.azagniotov.metrics.reporter.cloudwatch CloudWatchReporter]
+           [java.io Closeable]
+           [software.amazon.awssdk.services.cloudwatch CloudWatchAsyncClient]
```

Comment:
this isn't the official AWS SDK package, I don't think? it's usually `com.amazonaws.*`, `com.amazonaws.services.cloudwatch.AmazonCloudWatchAsyncClient` in this case

Reply:
ah, I stand corrected, this is for v2